### ISSUE TITLE
Text problems in Help

### ISSUE THREAD

**Issue by nist-aharvey** [usnistgov/REFPROP-issues#167]
This is mostly cosmetic, so low priority.
When I am in the Help, if I right-click on a link to see what it connects to and choose "Properties", the window that comes up has its text garbled and cut off.  I am pasting in a screen capture of such a Window.  This is not unique to any one link; it happens to all the ones I tried.
If it helps diagnose anything, TDE Help has a similar problem (actually it is worse in TDE, almost none of the text is visible there).
Sometimes things like this happen because my screen resolution is set at 125%.

![image](https://user-images.githubusercontent.com/45049145/58338042-53a80500-7e04-11e9-8101-555c6034dc61.png)

**Comment by ianhbell** [comment]
What if you use 100% zoom?  The tool for help files in windows is rather archaic, so I am not surprised that you have issues.  is it better if you use the online help : https://refprop-docs.readthedocs.io/en/latest/GUI/index.html ?

**Comment by nist-aharvey** [comment]
I tried changing to 100% and rebooting, and the problem went away (both for REFPROP and TDE).  So it must be that something in the Help tool isn’t scaling properly when I have it set at 125% (which my middle-aged eyes need).
The online Help (which I was not aware existed!) looks fine for me at 125%.  Of course there is no right-click Properties window that comes up for links in that environment, which is the only place I saw the problem in the GUI Help.
